content/ru/docs/concepts/workloads/_index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Рабочие нагрузки"
+weight: 55
+description: >
+  Поймите под, самый маленький развертываемый вычислительный объект в Kubernetes, и абстракции более высокого уровня, которые помогут вам их запускать.
+no_list: true
+card:
+  title: Рабочие нагрузки и поды
+  name: concepts
+  weight: 60
+---
+
+Рабочая нагрузка — это приложение, работающее в Kubernetes.
+Независимо от того, представляет ли ваша рабочая нагрузка один компонент
+или несколько, которые работают вместе, в Kubernetes вы 
+запускаете ее внутри набора [_подов_](/docs/concepts/workloads/pods).
+В Kubernetes под представляет собой набор работающих
+{{< glossary_tooltip text="контейнеров" term_id="container" >}} в вашем кластере.
```

Comment:
As a general recommendation for translations, it's better to avoid too many "вы" и "ваш". While they are essential in English, you might skip _most_ (yet not _all_) of them in Russian.
```suggestion
{{< glossary_tooltip text="контейнеров" term_id="container" >}} в кластере.
```

Reply:
Good point, I'll keep it in mind next time.